clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/lifecycle/InstanceTerminationLifecycleWorkerProvider.java

```diff
@@ -71,8 +71,7 @@
   @PostConstruct
   public void start() {
     NetflixAmazonCredentials credentials =
-        (NetflixAmazonCredentials)
-            accountCredentialsProvider.getCredentials(properties.getAccountName());
+        credentialsRepository.getOne(properties.getAccountName());
     ExecutorService executorService =
```

Comment:
I got an NPE running this locally.

The `properties.getAccountName()` should be a legitimate account, something at play with the @PostConstruct ordering?

Reply:
looking into it, thanks!